src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -414,6 +415,37 @@ protected void paintComponent(final Graphics g) {
     updateStep();
     uiContext.addShutdownWindow(this);
   }
+  
+  
+  public static KeyListener getFlagToggleKeyListener(TripleAFrame frame) {
+    return new KeyListener() {
+      private boolean blockInputs = false;
+      @Override
+      public void keyTyped(final KeyEvent e) {/*Do nothing*/}
+
+      @Override
+      public void keyPressed(final KeyEvent e) {
+        if(!blockInputs){
```

Comment:
Well in #684 you said:
> let's bind a hotkey that you can hold down to toggle the flags on/off

So I did what you said... ^^ If you prefer to have them on by default, you can check the checkbox to do so...